go.mod

```diff
@@ -164,7 +160,12 @@ require (
 )
 
 replace (
-	golang.org/x/lint => golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
```

Comment:
sorry that I have to raise the comment again.

We should put x/lint back, right? otherwise, the version will get downgrade from `v0.0.0-20210508222113-6edffad5e616` to `v0.0.0-20190313153728-d0100b6bd8b3`. It seems to me the most important conclusion is that we don't give up the newer version against which we've successfully tested.

Reply:
Good catch. Let's put that back.

Reply:
Sorry, let me fix it.